lib/ansible/modules/package.py

```diff
@@ -29,6 +29,7 @@
       - Package name, or package specifier with version.
       - Syntax varies with package manager. For example V(name-1.0) or V(name=1.0).
       - Package names also vary with package manager; this module will not "translate" them per distro. For example V(libyaml-dev), V(libyaml-devel).
+      - To operate on several packages this can accept a comma separated string of packages or a list of packages.
```

Comment:
this should just be `type: list` .. also x2 check that all package manager modules support a list.

Reply:
I copied the description from the `yum` / `dnf` modules.

Reply:
Of the list in https://github.com/ansible/ansible/blob/e80507af32fad1ccaa62f8e6630f9095fe253004/lib/ansible/module_utils/facts/system/pkg_mgr.py#L17-L49 `atomic_container`, `swdepot`, `svr4pkg`, and `swupd` do not appear to support lists.

Reply:
package is a 'generic' and real support comes from the underlying modules used, if not all of them allow name to be a list, we should not add that information here.

Reply:
in which case the example ought to be removed as that's inconsistent with above.

Reply:
or at least amended 'depending on package manager'

Reply:
that is a good solution. Done.